// Try Catch se utiliza en partes criticas cuando no podemos dejar q no funcione la aplicacion y 
//queremos probar si la linea de codigo funciona y a su vez que el demas codigo aparezca

const num1 = 10;
const num2 = 20;
const num3 = 30;

try {
    console.log(num4);
} catch (error) {
    console.log(error);
}


console.log(num1);

console.log(num2);
console.log(num3);